### DISCUSSION TITLE
Running AirSim in UE4.26

### DISCUSSION THREAD

**Discussion by drbswa** [microsoft/AirSim#4125] [Support Q&A] [answered]
I'm working on making a custom map with a few different terrain generation plugins, but I need to be on 4.26 for those to work properly. Is there anything special that needs to be done to run AirSim in 4.26?

**Comment by LittleTimeWxk** [comment, marked as answer]
There is nothing special work to do. But you should pay attention to the corresponding version of Visital Studio when using different versions of AirSim. The precautions are explained in the document when downloading the code.
